spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfigurationTest.java

```diff
@@ -260,10 +266,38 @@ void axonServerEventStoreFactoryBeanIsConfigured() {
     @Test
     void axonServerEventStoreFactoryBeanIsNotConfiguredWhenEventStoreIsDisabled() {
         testContext.withUserConfiguration(TestContext.class)
-                .withPropertyValues("axon.axonserver.event-store.enabled=false")
+                   .withPropertyValues("axon.axonserver.event-store.enabled=false")
                    .run(context -> assertThat(context).getBean(AxonServerEventStoreFactory.class).isNull());
     }
 
+    @Test
+    void subscribableMessageSourceDefinitionShouldBeConfigured() {
+        testContext.withPropertyValues("axon.axonserver.auto-persistent-streams-enable=true",
+                                       "axon.axonserver.auto-persistent-streams-settings.initial-segment-count=10",
+                                       "axon.axonserver.auto-persistent-streams-settings.batch-size=6")
+                   .run(context -> {
+
+                       EventProcessingModule eventProcessingModule = context.getBean(EventProcessingModule.class);
+
+                       EventProcessingConfigurer.SubscribableMessageSourceDefinitionBuilder builder = getField(
+                               "subscribableMessageSourceDefinitionBuilder",
+                               eventProcessingModule);
+
+                       assertThat(builder).isNotNull();
+
+                       SubscribableMessageSourceDefinition<EventMessage<?>> definition = builder.build(
+                               "processingGroupName");
+                       assertThat(definition).isInstanceOf(PersistentStreamMessageSourceDefinition.class);
+                       String name = getField("name", definition);
```

Comment:
Axon Framework has a `ReflectionUtils` utility class for this! But, feel free to keep your custom solution here. Just figured it's worth sharing that it exists.